docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -123,6 +123,14 @@ For many concepts, a reference document exists in the `reference/concepts` or `r
 
 See [this example file][example-reference-document-implementations].
 
+## Add or update story document
+
+**Purpose:** Allow maintainers to find out which tracks have implemented an exercise for a story.
+
+Each exercise has a story. These stories are documented in the `reference/stories` directory. If such a document exists for the exercise's story, add the exercise to the corresponding exercise's `Implementations` section. When no story document exists, consider adding one.
```

Comment:
```suggestion
Each exercise has a story or theme. The stories people have created so far are documented in the `reference/stories` directory. If such a document exists for the exercise's story, add the exercise to the corresponding exercise's `Implementations` section. When no story document exists, consider adding one.
```

Changed it slightly to make it feel more like a living thing, rather than a predefined list of stories. Happy either way though. Thanks for keeping the docs up to date! 💙

Reply:
Excellent suggestion!